content/services/consulting.md

```diff
@@ -147,7 +147,7 @@ for the people who visit.
   slugs=['pleasant', 'best-in-the-world']
 ) }}
 
-{{ layout.title('Services') }}
+{{ layout.title('Packages') }}
```

Comment:
Is this intentional?

Reply:
Yes, from: https://github.com/oddbird/oddleventy/pull/819#discussion_r1983850274